linera-client/src/client_context.rs

```diff
@@ -185,6 +185,41 @@ where
         }
     }
 
+    #[cfg(with_testing)]
+    pub fn new_test(storage: S, wallet: W) -> Self {
```

Comment:
`{create,make,new}_test_client_context` ?

Reply:
I thought the `client_context` suffix was redundant, since we're calling it from a `ClientContext` struct 🤔 but I can change it